<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0" colspan="5">
        <widget class="QPushButton" name="btnInitData">
         <property name="text">
          <string>初始化表格数据</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QCheckBox" name="checkBoxEdit">
         <property name="text">
          <string>表格可编辑</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="3" colspan="2">
        <widget class="QCheckBox" name="checkBoxRowColor">
         <property name="text">
          <string>间隔行底色</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QCheckBox" name="checkBoxRowHeader">
         <property name="text">
          <string>显示行表头</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="3" colspan="2">
        <widget class="QCheckBox" name="checkBoxColumnHeader">
         <property name="text">
          <string>显示列表头</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QRadioButton" name="radioButtonRowSelect">
         <property name="text">
          <string>行选择</string>
         </property>
        </widget>
       </item>
       <item row="9" column="3" colspan="2">
        <widget class="QRadioButton" name="radioButtonCellSelect">
         <property name="text">
          <string>单元格选择</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QPushButton" name="btnInsertRow">
         <property name="text">
          <string>插入行</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3" colspan="3">
        <widget class="QPushButton" name="btnAddRow">
         <property name="text">
          <string>添加行</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="6">
        <widget class="QPushButton" name="btnDeleteCurrentRow">
         <property name="text">
          <string>删除当前行</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="6">
        <widget class="QPushButton" name="btnReadRowToText">
         <property name="text">
          <string>读取表格内容到文本</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="6">
        <widget class="QPushButton" name="btnSetHeader">
         <property name="text">
          <string>设置表头</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4" colspan="2">
        <widget class="QSpinBox" name="spinBoxRow">
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="QPushButton" name="btnAutoHeight">
         <property name="text">
          <string>自动调节行高</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3" colspan="3">
        <widget class="QPushButton" name="btnAutoWight">
         <property name="text">
          <string>自动调节列宽</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QPushButton" name="btnSetRow">
         <property name="text">
          <string>设置行数</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="rowCount">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>200</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>805</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
